packages/application-to-register-webapp/src/routes/land/register-land-task-list.js

```diff
@@ -1,7 +1,14 @@
 import constants from '../../utils/constants.js'
+import REGISTER_TASK_LIST from '../../utils/register-task-list.js'
 
 const handlers = {
-  get: async (_request, h) => h.view(constants.views.REGISTER_LAND_TASK_LIST)
+  get: async (request, h) => {
+    let dataContent = request.yar.get(constants.redisKeys.REGISTRATION_TASK_DETAILS)
+    if (dataContent === null || dataContent === undefined) {
```

Comment:
Couldn't the null and undefined checks be replaced with a falsey statement. ie `if (!dataContent)`

And then could it not be reduced to `const dataContent = request.yar.get(constants.redisKeys.REGISTRATION_TASK_DETAILS) || REGISTER_TASK_LIST`

Reply:
Use a falsey